sdk/keyvault/keyvault-certificates/src/lro/core-lro-update.ts

```diff
@@ -175,7 +199,9 @@ export abstract class KVPoller<TState extends PollOperationState<TResult>, TResu
       const clearPollOncePromise = (): void => {
         this.pollOncePromise = undefined;
       };
+      /* eslint-disable promise/catch-or-return */
```

Comment:
instead of disable/enable you can make this comment

```
/* eslint-disable-next-line promise/catch-or-return */
```

and skip the re-enable

Reply:
Thanks! I forgot about that one 🌞